config/teleop.yaml

```diff
@@ -86,6 +86,15 @@ teleop:
     pan: 4
     tilt: 5
 
+  euclidean_error_threshold: 100
+
+  joint_error_thresholds:
+    joint_a: 100
+    joint_b: 100
+    joint_c: 100
+    joint_d: 100
+    joint_e: 100
```

Comment:
Can you add a description of units for these?

Reply:
Also are these reasonable? I thought this was in radians?

Reply:
I changed the individual thresholds to pi/2 and euclidean threshold to pi, but me and Joe don't know what good threshold values would be.

Reply:
Yeah I figured we would determine what good values look like once we start testing.